templates/base/src/pages/_document.js

```diff
@@ -10,7 +10,19 @@ class MyDocument extends Document {
   render() {
     return (
       <Html lang="en">
-        <Head />
+        <Head>
+          <meta name='application-name' content='Universal React V2' />
```

Comment:
This should be done via a component, so that conditionally we are able to add or skip it.